src/Concretize.hs

```diff
@@ -1,3 +1,4 @@
+{-# LANGUAGE BangPatterns #-}
```

Comment:
Seems like no BangPatterns are used?

Reply:
Good catch! I used this for debugging, but then didn’t need it in the final product.